### ISSUE TITLE
[limitation] Early stopping at seed 2026 need not show a validation rise

### ISSUE THREAD

**Issue by pavanamthomas** [pavanamthomas/pytorch-deep-learning-lab#1]
On the current toy split the last epoch can be the best epoch. That is allowed. It is not locked as a required overfit.
